src/components/Text/Text.stories.tsx

```diff
@@ -79,6 +79,23 @@ export const Overline: StoryObj<Args> = {
   },
 };
 
+export const Spacing: StoryObj<Args> = {
+  render: () => (
+    <div>
```

Comment:
initially had this as fragment, and the snap would only take the first `<Text>` child